reporter/src/main/kotlin/reporters/NoticeTemplateReporter.kt

```diff
@@ -177,6 +177,16 @@ class NoticeTemplateReporter : Reporter {
                 licenseConfiguration.licenses.find { it.id == resolvedLicense.license }?.includeInNoticeFile ?: true
             }
 
+        /**
+         * Filter licenses that are configured to require a
+         * [source code offer in the notice file][License.includeInNoticeFile] in the [LicenseConfiguration].
+         */
+        @Suppress("UNUSED") // This function is used in the templates.
```

Comment:
Would it make sense to make this an extension function of `List<ResolvedLicense>`?

Reply:
No, that would make the function harder to call from the template, because it does not support Kotlin extension functions.